man/io_uring_setup.2

```diff
@@ -148,6 +149,27 @@ The
 .I resv
 array must be initialized to zero.
 
+.I features
+is filled in by the kernel, which specifies various features supported
+by current kernel version.
+.TP
+.B IORING_FEAT_SINGLE_MMAP
+.TP
+.B IORING_FEAT_NODROP
+.TP
+.B IORING_FEAT_SUBMIT_STABLE
+If this flag is set, applications can be certain that any data for
+async offload has been consumed when the kernel has consumed the SQE.
+.TP
+.B IORING_FEAT_RW_CUR_POS
+If this flag is set, applications can specify offset == -1 to mean
```

Comment:
I think that first sentence should be:

If this flag is set, applications can specify offset == -1 with
.B IORING_OP_READV, IORING_OP_WRITEV, IORING_OP_READ,
and
.B IORING_OP_WRITE
to mean [...]

and so on.